common/appscale/common/retrying.py

```diff
@@ -26,9 +31,135 @@ def not_missed(value_1, value_2):
   raise BothMissedException()
 
 
+class BackoffSequence(object):
+  """
+  Iterable sequence of backoff delays.
+  This class suppose to be in cases when you need to do retries.
+  
+  Usage examples (find more in common/test/unit/test_retrying.py):
+  
```

Comment:
It might be helpful to highlight what the defaults are so that when someone inspects the object via dir() or help() they can quickly see what the default attributes are. For example with <span>#</span>1 I don't know how many times it will retry the operation by default.